pricegraph/src/orderbook/order.rs

```diff
@@ -29,12 +56,51 @@ impl OrderMap {
     }
 
     /// Returns an iterator over the collection of orders for each token pair.
-    pub fn all_pairs_mut(&mut self) -> impl Iterator<Item = (TokenPair, &'_ mut Vec<Order>)> + '_ {
+    fn all_pairs_mut(&mut self) -> impl Iterator<Item = (TokenPair, &'_ mut Vec<Order>)> + '_ {
         self.0.iter_mut().flat_map(|(&sell, o)| {
             o.iter_mut()
                 .map(move |(&buy, o)| (TokenPair { sell, buy }, o))
         })
     }
+
+    /// Returns the orders for an order pair. Returns `None` if that pair has
+    /// no orders.
+    fn pair_orders(&self, pair: TokenPair) -> Option<&[Order]> {
```

Comment:
naming nit: since pair is also a verb it confused me in the beginning. Maybe `orders_for_pair` is more clear?